dev/tests/js/jasmine/tests/app/code/Magento/Ui/base/js/grid/data-storage.test.js

```diff
@@ -401,73 +309,83 @@ define([
                 dataScope: 'magento'
             });
 
-            it('check "model._requests"', function () {
+            it('adds the request to the cache', function () {
                 var params = {
-                    namespace: 'magento',
-                    search: '',
-                    sorting: {},
-                    paging: {}
-                },
-                data = {
-                    items: ['1','2','3'],
-                    totalRecords: 3
-                };
+                        namespace: 'magento',
+                        search: '',
+                        sorting: {},
+                        paging: {}
+                    },
+                    ids = ['1','2','3'],
+                    data = {
+                        items: ids,
+                        totalRecords: 3,
+                        errorMessage: ''
+                    },
+                    request = {
+                        ids: ids,
+                        params: params,
+                        totalRecords: 3,
+                        errorMessage: ''
+                    };
 
                 spyOn(model, 'removeRequest');
-                spyOn(model, 'getIds').and.callFake(function () {
-                    return ['1','2','3'];
-                });
+                spyOn(model, 'getIds').and.returnValue(ids);
                 model.cacheRequest(data, params);
-                expect(typeof model._requests).toEqual('object');
-                expect(model.getIds).toHaveBeenCalled();
+                expect(model.getRequest(params)).toEqual(request);
```

Comment:
I think here we need to also test `model._requests`

Reply:
I think it's better to test it through `getRequest` as `_requests` property is a detail of internal implementation

Reply:
Okay, `_requests` array pushed functionality is added in the `cacheRequest` method so I thought that we need to cover in the `cacheRequest` functionality